rename resource module to asset module

DominikB2014 opened the pull request:
Asset module probably resonates better with FE devs, so we should rename the module

lizokm reviewed (approved):
LGTM!

bcoe commented on docs/product/performance/assets.mdx:
```diff
@@ -1,20 +1,20 @@
 ---
 
-If you have performance monitoring enabled for your frontend, you can see how your browser resources are performing in Sentry.
+If you have performance monitoring enabled for your frontend, you can see how your browser assets are performing in Sentry.
```
Thanks for doing this so fast @DominikB2014 🎉 

Mind adding a banner that mentions we've renamed this page, something like:

> Note: This product feature was originally named "Resources", "Assets" has the same functionality with a new _exciting_ name.

Maybe someone else can come up with a more humorous blurb that gets the point across.

DominikB2014 replied:
Yeah, will do!

vivianyentran commented on docs/product/performance/assets.mdx:
```diff
@@ -118,15 +118,15 @@ The following tokens will be replaced with \* within a resource url
-## Resource Summary Page
+## Asset Summary Page
 
 To open a resource's **Resource Summary** page, click on the resource from the table in the **Resources** page.
```
This still mentions resource instead of assets, did you mean to replace with assets for the rest of the page too?

DominikB2014 replied:
Yeah this should be assets, good catch! , I'll have to go through this to double it all. There are some cases we need to keep the term Resources (like mentioning span ops), so I cant do a replaceall

vivianyentran reviewed (changes requested):
Changes look fine but you'll want to double check the following before merging:

- Add redirect for this page to middleware.ts (see https://docs.sentry.io/contributing/pages/redirects/#add-a-new-redirect-in-middlewarets-recommended)
- Are the widgets affected by this change? Might need to update these sections:
- https://docs.sentry.io/product/performance/filters-display/widgets/#most-time-consuming-resources
- https://docs.sentry.io/product/performance/filters-display/widgets/#slow-resource-ops
- https://docs.sentry.io/product/performance/filters-display/#actionable-widgets
- Since you're changing the file name, will also want to do a search to update any links that were going directly to the resources page (for example, links here https://docs.sentry.io/product/performance/filters-display/widgets/#most-time-consuming-resources)

We should also update the arcades included in these docs. I can help with that; just give me a heads up for when this change is planning to launch!

DominikB2014 commented:
@vivianyentran  we plan to release this on June 5th, r we able to update the arcades by then? Also, do the links need be changed?

vivianyentran commented:
@DominikB2014 Yes, we should be able to update the arcades by then. Are all the changes live in the sentry org to capture screenshots?

vivianyentran commented:
@DominikB2014 Yes, you should also update any links that point to the old path

DominikB2014 commented:
> @DominikB2014 Yes, we should be able to update the arcades by then. Are all the changes live in the sentry org to capture screenshots?

@vivianyentran the updates should be live eod today, I'll follow up with you on slack when it's done!

DominikB2014 commented:
@vivianyentran 
> * Add redirect for this page to middleware.ts (see https://docs.sentry.io/contributing/pages/redirects/#add-a-new-redirect-in-middlewarets-recommended)

Added redirects!
> * Are the widgets affected by this change? Might need to update these sections:
> * https://docs.sentry.io/product/performance/filters-display/widgets/#most-time-consuming-resources

Yes, just updated the screenshot for this widget
> * https://docs.sentry.io/product/performance/filters-display/widgets/#slow-resource-ops

No, this one stays the same!
> * https://docs.sentry.io/product/performance/filters-display/#actionable-widgets
Updated the link!

> * Since you're changing the file name, will also want to do a search to update any links that were going directly to the resources page (for example, links here https://docs.sentry.io/product/performance/filters-display/widgets/#most-time-consuming-resources)
Looked through, and not seeing any more links to be updated

vivianyentran commented on docs/product/performance/assets.mdx:
```diff
@@ -1,20 +1,24 @@
-Starting with the [**Resources** page](#resources-page), you get a high-level overview of how your resources are doing. From there, you can drill into a specific resource's [**Resource Summary** page](#resource-summary-page) and then investigate sample events from the [Sample List](#sample-list) to better understand the context of its performance in a specific page.
+Starting with the [**Assets** page](#assets-page), you get a high-level overview of how your assets are doing. From there, you can drill into a specific asset's [**Asset Summary** page](#asset-summary-page) and then investigate sample events from the [Sample List](#sample-list) to better understand the context of its performance in a specific page.
 
 <div style={{position: 'relative', paddingBottom: 'calc(50.86013462976814% + 41px)', height: 0}}><iframe src="https://demo.arcade.software/BxySgGk7hpJ72xquLdKe?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style={{position: 'absolute', top: 0, left: 0, width: '100%', height: '100%', colorScheme: 'light'}}></iframe></div>
```
```suggestion
<div style={{position: 'relative', paddingBottom: 'calc(50.86013462976814% + 41px)', height: 0}}><iframe src="https://demo.arcade.software/c7lgN0cFkg0R2tb9t0jH?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style={{position: 'absolute', top: 0, left: 0, width: '100%', height: '100%', colorScheme: 'light'}}></iframe></div>
```
Updated the arcade link! Will remove the other arcades as this one covers the entire workflow (going to do the same for the other modules as well in a separate PR)

DominikB2014 replied:
Perfect!

vivianyentran commented on docs/product/performance/assets.mdx:
```diff
@@ -72,42 +76,41 @@ resources.forEach((entry) => {
-To view more details, click on a resource from the table to open its **Resource Summary** page.
+To view more details, click on a asset from the table to open its **Asset Summary** page.
 
 <div style={{position: 'relative', paddingBottom: 'calc(50.86013462976814% + 41px)', height: 0}}><iframe src="https://demo.arcade.software/dVvM0eWdGjN7bVhxSwJr?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style={{position: 'absolute', top: 0, left: 0, width: '100%', height: '100%', colorScheme: 'light'}}></iframe></div>
```
```suggestion
```

vivianyentran commented on docs/product/performance/assets.mdx:
```diff
@@ -118,43 +121,37 @@ The following tokens will be replaced with \* within a resource url
 
 If you want to investigate a specific page, click on it to open a sidebar showing some sample events.
 
 <div style={{position: 'relative', paddingBottom: 'calc(50.829268292682926% + 41px)', height: 0}}><iframe src="https://demo.arcade.software/D96Pu2LMLHddyVCn3BOd?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style={{position: 'absolute', top: 0, left: 0, width: '100%', height: '100%', colorScheme: 'light'}}></iframe></div>
```
```suggestion
```

vivianyentran commented on docs/product/performance/assets.mdx:
```diff
@@ -164,6 +161,6 @@ These same events are also represented as triangles shown in the average duratio
-From the sample list, you can drill down to specific good, average, or bad examples of a given resource within a given page. Click on an event ID to drill into the resource's span details within the span waterfall of the [Trace Details](/concepts/key-terms/tracing/trace-view/#product-walkthrough-trace-details-page) page.
+From the sample list, you can drill down to specific good, average, or bad examples of a given asset within a given page. Click on an event ID to drill into the asset's span details within the span waterfall of the [Trace Details](/concepts/key-terms/tracing/trace-view/#product-walkthrough-trace-details-page) page.
 
 <div style={{position: 'relative', paddingBottom: 'calc(50.829268292682926% + 41px)', height: 0}}><iframe src="https://demo.arcade.software/h2B547DqgJGIrlsMeVWx?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style={{position: 'absolute', top: 0, left: 0, width: '100%', height: '100%', colorScheme: 'light'}}></iframe></div>
```
```suggestion
```
